src/Document.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace OpenFoodFacts;
+
+use OpenFoodFacts\Document\FoodProduct;
+use OpenFoodFacts\Document\PetProduct;
+use OpenFoodFacts\Document\BeautyProduct;
+
+/**
+ * In mongoDB all element are object, it not possible to define property.
+ * All property of the mongodb entity are store in one property of this class and the magic call try to access to it
+ */
+class Document
+{
+    /**
+     * the whole data
+     * @var array
+     */
+    private $data;
+
+    /**
+     * the whole data
+     * @var array
+     */
+    private $api;
+
+    /**
+     * Initialization the document and specify from which API it was extract
+     * @param array $data the whole data
+     * @param array $api the api name
```

Comment:
typo mistake here 😉 : is string, isn't it ?

Reply:
Yes.